src/main/webapp/src/app/hidden-gem.service.spec.ts

```diff
@@ -110,35 +96,28 @@ describe('HiddenGemService', () => {
     const expectedTopGems: HiddenGem[] =
```

Comment:
Maybe we don't need this test anymore. We initially had a 2 tests for the http call for ranking and another for recommendations. Now that its all handled in the one hidden gems call, I feel like this test might be redundant. It's basically testing the same thing as the first one?

Reply:
That's a good point! I removed the 2 duplicate tests :)